modules/sdk-core/src/bitgo/utils/mpcUtils.ts

```diff
@@ -132,6 +138,10 @@ export abstract class MpcUtils {
             'token type and quantity is required to request a transaction with intent to transfer a token'
           );
         }
+        tokenData.tokenName = this.getTokenName(baseCoin, tokenData);
```

Comment:
to get the NFT collection name from statics using the NFT collection ID present in `tokenData.tokenContractAddress`.